### PR TITLE
[devtools-translate] Japanese - What's New in DevTools (Chrome 106)

### LINKED ISSUE #3733: [devtools-translate] Japanese - What's New in DevTools (Chrome 106)
Post: https://developer.chrome.com/blog/new-in-devtools-106/
Translated by: Sep 23

Translator: @yoichiro
Reviewers: @yoshiko-pg, @technohippy, @lacolaco

### PR DESCRIPTION
**Pull request by yoichiro**
Fixes #3733

### CODE REVIEW

**Comment by yoichiro** [comment]
@technohippy @lacolaco @yoshiko-pg Could you review this pull request? Thanks in advance!

**Comment by yoichiro** [comment]
@jecfish Could you add labels `devtools-translate` and `translation` to this pull request?

**Comment by jecfish** [comment]
Yes I can, done! Thanks for the translations. over to the team for review. :)

File: site/ja/blog/new-in-devtools-106/index.md [hunk 1]
```diff
@@ -15,39 +15,39 @@ tags:
 draft: true
```
Comment (jecfish) [inline site/ja/blog/new-in-devtools-106/index.md:15]:
```suggestion
```
Remove draft

**Review by jecfish — CHANGES_REQUESTED** [review]

File: site/ja/blog/new-in-devtools-106/index.md (hunk 1 above)
Comment (yoichiro) [reply site/ja/blog/new-in-devtools-106/index.md]: @jecfish Oh, thank you for the suggestion!

File: site/ja/blog/new-in-devtools-106/index.md [hunk 2]
```diff
@@ -59,76 +58,96 @@ async increment() {
...
 
 <!-- Behind the scenes, DevTools introduced a new “Async Stack Tagging” feature. You can tell the whole story of the operation by linking both parts of the async code together with the new `console.createTask()` method. See [Modern debugging in DevTools](/blog/devtools-modern-web-debugging/#linked-stack-traces) to learn more.  -->
+舞台裏では、DevTools に新しい "Async Stack Tagging" 機能が導入されました。非同期コードの両方の部分を新しい `console.createTask()` メソッドと一緒にリンクすることで、処理の全体像を伝えることができます。詳細については、 [Modern debugging in DevTools](/blog/devtools-modern-web-debugging/#linked-stack-traces) を参照してください。
 
 <!-- Does it sound complicated? Not at all. Most of the time, the framework you are using handles the scheduling and async execution. In that case, it is up to the framework to implement the API, you don’t need to worry about it. (e.g. Angular implemented these [changes](https://chromium-review.googlesource.com/c/v8/v8/+/3776678)) -->
+複雑に聞こえますか？全くそんなことはありません。ほとんどの場合、使用しているフレームワークがスケジューリングと非同期実行を処理します。その場合、API を実装するのはフレームワーク次第なので、心配する必要はありません。 (例: Angular はこれらの [変更]((https://chromium-review.googlesource.com/c/v8/v8/+/3776678)) を実装しました)
```
Comment (yoshiko-pg) [inline site/ja/blog/new-in-devtools-106/index.md:72]:
```suggestion
複雑に聞こえますか？全くそんなことはありません。ほとんどの場合、使用しているフレームワークがスケジューリングと非同期実行を処理します。その場合、API を実装するのはフレームワーク次第なので、心配する必要はありません。 (例: Angular はこれらの [変更](https://chromium-review.googlesource.com/c/v8/v8/+/3776678) を実装しました)
```

**Review by yoshiko-pg — COMMENTED** [review]
おつかれさまです！ LGTM!!

File: site/ja/blog/new-in-devtools-106/index.md (hunk 2 above)
Comment (yoichiro) [reply site/ja/blog/new-in-devtools-106/index.md]: @yoshiko-pg おっと、ありがとうございます！

**Review by lacolaco — COMMENTED** [review]
違和感なく読めました！ LGTMです

**Comment by yoichiro** [comment]
@jecfish @agektmr  We discussed this review process and decided that we change the process to the following:

* If "two" reviewers judge as LGTM, the request is approved.

We would like to apply the rule from this pull request. This pull request has already gotten LGTMs from two reviewers. Jecelyn, could you final check and merge this pull request to the main branch if there is nothing wrong?

**Comment by jecfish** [comment]
Sound good for two reviewers LGTM. Will take note of that. 

Thanks all for the translation and reviews. Merged!

**Review by jecfish — APPROVED** [review]

**Pull request merged**